### PR TITLE
Feature/daef 93 language selection

### PR DESCRIPTION
**Pull request by nikolaglumac**
This PR introduces page for selecting user interface language on application launch.

### CODE REVIEW

File: app/i18n/locales/ko_KR.json [hunk 1]
```diff
@@ -16,6 +16,8 @@
   "loading.screen.connectingToNetworkMessage": "!!!Connecting to network",
   "loading.screen.reconnectingToNetworkMessage": "!!!Network connection lost - reconnecting",
   "loading.screen.syncingBlocksMessage": "!!!Syncing blocks",
+  "profile.languageSelect.form.languageSelectLabel": "!!!Select your language",
+  "profile.languageSelect.form.submitLabel": "!!!Continue",
```
Comment (darko-mijic) [inline app/i18n/locales/ko_KR.json]: There is no need for adding translations manually except when entering real translations. Keys with default translations are managed by `npm run manage-translations`

File: app/stores/AppStore.js [hunk 2]
```diff
@@ -1,19 +1,53 @@
...
+    this.actions.profile.updateLocale.listen(this._updateLocale);
+    this._redirectToLanguageSelectionIfNoLocaleSet();
   }
 
   @computed get currentRoute(): string {
     return this.stores.router.location.pathname;
   }
 
+  @computed get currentLocale(): string {
+    if (!this.profileLocaleRequest.wasExecuted) {
```
Comment (DominikGuzei) [inline app/stores/AppStore.js]: Please re-use the `isCurrentLocaleSet` method here 😉

File: app/stores/AppStore.js [hunk 3]
```diff
@@ -1,19 +1,53 @@
...
+    }
+    return this.profileLocaleRequest.result;
+  }
+
+  @computed get isCurrentLocaleSet(): string {
+    return this.profileLocaleRequest.wasExecuted;
+  }
+
+  @action _redirectToLanguageSelectionIfNoLocaleSet = () => {
+    if (!this.profileLocaleRequest.wasExecuted)  {
```
Comment (DominikGuzei) [inline app/stores/AppStore.js]: Please re-use the `isCurrentLocaleSet` method here 😉

File: app/i18n/locales/ko_KR.json (hunk 1 above)
Comment (nikolaglumac) [inline app/i18n/locales/ko_KR.json]: @darko-mijic thanks for pointing this out! All manually added translation keys are now removed.

File: app/stores/AppStore.js (hunk 3 above)
Comment (nikolaglumac) [inline app/stores/AppStore.js]: @DominikGuzei good point! Fixed it in my lastest commit ;)

**Comment by DominikGuzei** [comment]
@nikolaglumac im getting invalid key file imported error when running the tests:
screenshot 2017-03-21 um 22 06 05

**Comment by nikolaglumac** [comment]
@DominikGuzei I did not add new default-wallet.key in this branch so I guess this is the reason for the error. I will push updated key - the one from this commit: https://github.com/input-output-hk/daedalus/pull/170/commits/f7d7fa134c373b8508e5fdf3d746f60548dddc78

**Comment by DominikGuzei** [comment]
@nikolaglumac ok great 👍

**Comment by nikolaglumac** [comment]
@DominikGuzei here you go :)

**Comment by darko-mijic** [comment]
Reviewing this now

**Pull request merged**